database/clickhouse/clickhouse.go

```diff
@@ -127,6 +127,10 @@ func (ch *ClickHouse) init() error {
 		ch.config.MultiStatementMaxSize = DefaultMultiStatementMaxSize
 	}
 
+	if len(ch.config.MigrationsTableEngine) == 0 {
```

Comment:
Is the odd white spacing a github issue?

Reply:
I would think so, `gofmt -s -w database/clickhouse/clickhouse.go` yield no diff.